src/components/list/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { List as ChakraList, ListItem } from '@chakra-ui/react';
+
+const List: FC<PropsWithChildren> = ({ children }) => {
+  const isSingleChild = !Array.isArray(children);
+  const childrenArray = isSingleChild ? [children] : children;
+
+  return (
+    <ChakraList spacing="md">
```

Comment:
Could we export UnorderedList, OrderedList and ListIcon too? The component as it is seems much too restrictive. In the new PDP, we are going to have icons and bullet points. 
https://www.figma.com/file/QYphttx4oaLD2Sd2L50meg/AS24-Detail?node-id=2821%3A135748

Reply:
So this change is intended to implement the "Simple List" as designed in Figma here https://www.figma.com/file/CHU4fpMfXpiAW4qNFzi9y1/11.-AS24---Web---Components?node-id=2101%3A53892
My understanding was that we create the components as we need them for our designs and the teams extend them according to their needs when they work on a feature that requires a more complex version of the component.
Do I have something wrong in mind / should we change that approach?

The right-hand side likely needs more discussion as a list does not have interaction states like hovers and so on as they designed that would be more like an open menu.